optical_flow:
    ros__parameters:
        timer_period: 0.01 # min 0.01
        sensor_timeout: 0.5
        parent_frame: odom
        child_frame: base_link
        x_init: 0.0
        y_init: 0.0
        z_height: 0.025
        board: pmw3901 # [pmw3901, paa5100]
        scaler: 5
        spi_nr: 0
        spi_slot: front # [front, back]
        rotation: 0 # [0, 90, 180, 270]
        publish_tf: false
